src/components/modals/AddGroupModal.vue

```diff
@@ -267,6 +275,18 @@ const submit = async () => {
     parentOrgId: toRaw(parentDistrict.value)?.id,
   };
 
+  const { data: orgNameExists } = await doesOrgNameExist();
+
+
+  if (orgNameExists) {
+    return toast.add({
+      severity: 'warn',
+      summary: 'Data Error',
+      detail: `${orgName.value} already exists.`,
+      life: TOAST_DEFAULT_LIFE_DURATION,
```

Comment:
Let's change this to an error severity since it blocks you from creating a group. And the summary is something like "Group Creation Error". Then in the detail, let's mention what type of group it was. For example, "Site with name ____ already exists. Site names must be unique.". 

I think a warning would be something like, you can still do what you want, but there may be issues.